src/manager.cpp

```diff
@@ -106,14 +107,30 @@ sdbusplus::async::task<> Manager::parseConfiguration()
     co_return;
 }
 
+sdbusplus::async::task<> Manager::processPendingNotifications()
+{
+    lg2::info("processing pending notification requests");
+
+    for (const auto& path : fs::directory_iterator(NOTIFY_SERVICES_DIR))
+    {
+        // TODO:  Make it co routine if needs
```

Comment:
I think we should send notifications in parallel by identifying unique services, ensuring each one is notified only once.